sschema/handler/include.py

```diff
@@ -16,22 +16,16 @@
 
 def _handler(include_paths, uri):
     '''Handles an URI of form include://'''
-    base = 'include://'
-    assert(uri.startswith(base))
-    uri = uri[len(base):]
-
-    data = None
-    for path in include_paths:
-        uri = os.path.join(path, uri)
-        try:
-            f = open(uri, 'r')
-        except FileNotFoundError:
-            continue
-        else:
-            with f:
-                data = yaml.load(f)
-                break
-    return data
+    scheme = 'include://'
+    assert(uri.startswith(scheme))
+    uri = uri[len(scheme):]
+
+    for base in include_paths:
+        path = os.path.join(base, uri)
+        if os.path.isfile(path):
```

Comment:
Should use the try-except-else method... otherwise you have a race condition and crash if the path is deleted after os.path.isfile returns and before you open it. Yes, this is unlikely, but we may as well write it in an atomic way.

Reply:
Fixed by c6f5bbf

Reply:
Fixed by c6f5bbf